.circleci/docker/Dockerfile.ubuntu1904

```
@@ -21,7 +21,7 @@
 #
 # (c) 2016-2019 solidity contributors.
 #------------------------------------------------------------------------------
-FROM buildpack-deps:disco
+FROM buildpack-deps:disco as base
```

```suggestion
FROM buildpack-deps:disco AS base
```
as a matter of homogeneity, I'd propose to have all keywords capitalized. (including the "as" a few lines below)